gk/main.c

```diff
@@ -707,15 +707,15 @@ gk_stage1(void *arg)
 		if (ret < 0) {
 			RTE_LOG(ERR, GATEKEEPER, "gk: cannot assign an RX queue for the front interface for lcore %u\n",
 				lcore);
-			return -1;
+			goto cleanup;
```

Comment:
Shouldn't the `return -1;` before the `for` statement become a `goto cleanup`? Notice that I cannot comment there since the code wasn't changed there.